dev-db/etcd/etcd-0.4.8.ebuild

```diff
@@ -37,14 +37,14 @@ src_compile() {
 }
```

Comment:
`!etcd_protocols_2? ( !dev-db/etcd:2 )` needs to be updated to conflict with etcd-starter instead.